collections/_article/dev-snapshot-godot-4-5-dev-1.md

```diff
@@ -0,0 +1,144 @@
+- 3D: Fix `Camera3D` gizmo representation to accurately reflect FOV ([GH-101884](https://github.com/godotengine/godot/pull/101884)).
+- 3D: Use physical keys for the Q/W/E/R editor shortcuts ([GH-103533](https://github.com/godotengine/godot/pull/103533)).
+- Animation: Support hiding functions calls in Method Tracks ([GH-96421](https://github.com/godotengine/godot/pull/96421)).
+- Core: Add `scene_changed` signal to `SceneTree` ([GH-102986](https://github.com/godotengine/godot/pull/102986)).
+- Core: Add DDS image load and save functionality ([GH-101994](https://github.com/godotengine/godot/pull/101994)).
+- Core: Don't duplicate internal nodes ([GH-89442](https://github.com/godotengine/godot/pull/89442)).
+- Core: Implement `get_size` and `get_access_time` methods to `FileAccess` ([GH-83538](https://github.com/godotengine/godot/pull/83538)).
+- Debugger: Allow locating VRAM resource by double-clicking ([GH-103949](https://github.com/godotengine/godot/pull/103949)).
+- Documentation: Overhaul `Node3D` documentation ([GH-87440](https://github.com/godotengine/godot/pull/87440)).
+- Editor: Add option to copy a file's name in the FileSystem dock ([GH-96536](https://github.com/godotengine/godot/pull/96536)).
+- Editor: Allow ignoring debugger error breaks ([GH-77015](https://github.com/godotengine/godot/pull/77015)).
+- Editor: Don't save unnecessarily with `save_before_running` ([GH-90034](https://github.com/godotengine/godot/pull/90034)).
+- Editor: Improve drag and drop into array property editors ([GH-102534](https://github.com/godotengine/godot/pull/102534)).
+- Editor: Replace `UID` and Surface upgrade tools with universal one ([GH-103044](https://github.com/godotengine/godot/pull/103044)).
+- Export: Android: Convert `compress_native_libraries` to a basic export option ([GH-104301](https://github.com/godotengine/godot/pull/104301)).
+- GDExtension: Include precision in extension_api.json ([GH-103137](https://github.com/godotengine/godot/pull/103137)).
+- GDScript: Highlight warning lines in Script editor ([GH-102469](https://github.com/godotengine/godot/pull/102469)).
+- GUI: Implement properties that can recursively disable child controls' `FocusMode` & `MouseFilter` ([GH-97495](https://github.com/godotengine/godot/pull/97495)).
+- GUI: Improve ColorPicker picker shape keyboard and joypad accessibility ([GH-99374](https://github.com/godotengine/godot/pull/99374)).
+- Import: Use `UID` in addition to path for extracted meshes, materials and animations ([GH-100786](https://github.com/godotengine/godot/pull/100786)).
+- Particles: Add emission shape gizmos to `Particles2D` ([GH-102249](https://github.com/godotengine/godot/pull/102249)).
+- Particles: Fix particle jitter when scene tree is paused ([GH-95912](https://github.com/godotengine/godot/pull/95912)).
+- Physics: Chunk tilemap physics ([GH-102662](https://github.com/godotengine/godot/pull/102662)).
```

Comment:
If we still consider that highlight and list are mutually exclusive, this one can be removed.

Lately I've been wondering whether we should still include the highlighted PRs in this list though.